src/main/ruby/core/transcoding.rb

```diff
@@ -97,7 +97,8 @@ def self.asciicompat_encoding(string_or_encoding)
 
       return unless transcoders and transcoders.size == 1
 
-      Encoding.find transcoders[0].to_s
+      enc = Encoding.find transcoders[0].to_s
```

Comment:
Minor, but maybe just rebase this into the commit that rewrote `Encoding::Converter#asciicompat_encoding`.

Reply:
I wanted to separate to highlight the change and keep the previous commit as pure refactoring with no semantic change. The logic before didn't contain this check.

Reply:
Sounds good.